testutils/clustermanager/gke.go

```diff
@@ -125,34 +123,37 @@ type GKESDKClient struct {
 
 func (gsc *GKESDKClient) create(project, location string, rb *container.CreateClusterRequest) (*container.Operation, error) {
 	parent := fmt.Sprintf("projects/%s/locations/%s", project, location)
+	if zoneFromLoc(location) != "" {
```

Comment:
Probably doesn't matter much, since this is invoked O(1) times per day, but the `paretn`, or `clusterFullPath` below should only be computed if `if` evaluated to false.